src/check_datapackage/check.py

```diff
@@ -95,7 +95,7 @@ def _create_explanation(issue: Issue) -> str:
     property_name = issue.jsonpath.removesuffix("$").split(".")[-1]
     number_of_carets = len(str(issue.instance))
     return (  # noqa: F401
-        f"At package{issue.jsonpath.removeprefix('$')}:\n"
+        f"At {issue.jsonpath.removeprefix('$.')}:\n"
```

Comment:
```suggestion
        f"At `{issue.jsonpath.removeprefix('$.')}`:\n"
```

very small thing, but it helps to have something that visually indicates what this is or isn't.

Reply:
I made a comment about possible rich styles for the JSON path in
https://github.com/seedcase-project/check-datapackage/issues/211#issuecomment-3645568749. I think it is a separate concept that could be part of another PR?

I also opened https://github.com/seedcase-project/check-datapackage/issues/250 to discuss the need for fallback markup.